.github/workflows/check.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, macOS, Windows]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
could you add `3.11` in the `pyproject.toml` file also?

https://github.com/mdsol/mauth-client-python/blob/542857531084eb9da937bcdbc62c9bf29f78ecd9/pyproject.toml#L19